Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorMaintainerCommand.java

```diff
@@ -24,16 +39,36 @@ public void initialize() {
         log.info("Initializing");
         if (!elevator.isCalibrated()) {
             log.warn("ELEVATOR UNCALIBRATED - THIS COMMAND WILL NOT DO ANYTHING!");
+            startTime = Timer.getFPGATimestamp();
+            attemptTime = startTime + calibrationAttemptTime.get();
         }
     }
 
     @Override
     public void execute() {
+        boolean maintain = false;
```

Comment:
Let's replace the three booleans with an enum - we'll talk more about it on Tuesday, but I suggest you do some reading on enums in the meantime.

Reply:
done